File: Java/the_math_class/src/the_math_class/tmc.java
package the_math_class;

public class tmc {

	public static void main(String[] args) {
		
		// Comparing
		double x =3.14;
		double y = -10;
		double z = Math.max(x, y);
		// This displays x (3.14) because it is the bigger number
		System.out.println(z);
		
		// Absolute value
		double a = 69;
		double b = -420;
		// Displays 28980 because the absolute value of 69*-420=28980
		double c = Math.abs(a * b);
		System.out.println(c);
		
		// Square root
		double d = 123;
		double e = 456;
		// Displays 236.8 because that's the square root of d (123) * e (456).
		double f = Math.sqrt(d * e);
		System.out.println(f);
		
		// Rounding
		
		// Normal rounding
		double h = 78.9;
		double i = 10.1112;
		// Displays 8 because that is h (78.9) / i (10.1112) rounded.
		double j = Math.round(h / i);
		System.out.println(j);
		
		// Round up
		double k = 6.9;
		double l = 4.20;
		// Displays 12 because that is k + l rounded up
		double m = Math.ceil(k + l);
		System.out.println(m);
		
		// Round down
		double n = 6.9;
		double o = 4.20;
		// Displays 2 because that is n - o rounded down
		double p = Math.floor(n - o);
		System.out.println(p);

	}

}

File: C#/The math class/tmc.cs
using System;
using System.Security.Principal;

namespace The_math_class;

class The_math_class
{
    static void Main(string[] args)
    {   
        // Power
        double x = 3;
        double a = Math.Pow(x, 2);
        //Outputs 9
        Console.WriteLine(a);
        
        // Square root
        double s = 100;
        double r = Math.Sqrt(s);
        // Outputs 10
        Console.WriteLine(10);
        
        // Absoulute value
        a = -5;
        double v = Math.Abs(a);
        // Outputs 5
        Console.WriteLine(v);
        
        // Rounding
        
        /*
         * Round (Number above 5 rounds up, below 5 rounds down)
         * Ceil(ing) (Rounds up)
         * Floor (Rounds down)
         */
        
        // Round
        r = 34.6;
        double d = Math.Round(r);
        // Outputs 35
        Console.WriteLine(d);
        
        //Ceil(ing)
        double c = 34.6;
        double l = Math.Ceiling(c);
        // Outputs 35
        Console.WriteLine(l);
        
        //Floor
        double f = 34.6;
        double floor = Math.Floor(f);
        // Outputs 34
        Console.WriteLine(floor);
        
        // Comparing
        
        /*
         * Max (highest value)
         * Min (Lowest value)
         */
        
        // Max
        double m = 1;
        a = 10;
        x = Math.Max(m , a);
        // Outputs 10
        Console.WriteLine(x);
        
        // Min
        m = 1;
        a = 10;
        x = Math.Min(m , a);
        // Outputs 1
        Console.WriteLine(x);


    }
}